README.md

```diff
@@ -41,20 +41,18 @@ brew install ninja
 ```
 
 If you have Android Studio installed, make sure `$ANDROID_NDK` is available.
-`ANDROID_NDK=/Users/username/Library/Android/sdk/ndk-bundle` for instance.
+`ANDROID_NDK=/Users/username/Library/Android/sdk/ndk/<version>` for instance.
```

Comment:
Compiling Skia for Android it looks for `$ANDROID_NDK/sources` which on my computer is at this path

<img width="893" alt="Screenshot 2022-07-15 at 00 46 19" src="https://user-images.githubusercontent.com/11232797/179100846-64fe5ddf-b3e1-46ad-8090-8b57ab1a9168.png">